src/main/java/stsjorbsmod/cards/wanderer/Firebolt.java

```diff
@@ -30,7 +30,8 @@
 
     private static final int COST = 1;
     private static final int DAMAGE = 8;
-    private static final int AMOUNT_PER_CLARITY = 2;
+    private static final int AMOUNT_PER_CLARITY = 1;
+    private static final int UPGRADE_PLUS_PER_CLARITY = 1;

```

Comment:
I think this needs a more significant update to remove `calculateBonusBaseDamage` and remove the `if(upgraded)` conditional from the burning effect in `use()`

Reply:
Yes it does. Oy.